.travis.yml

```diff
@@ -11,15 +11,21 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
+matrix:
+  allow_failures:
+    - 7.0
+
 services: mongodb
 
 before_install:
-    - "[ $TRAVIS_PHP_VERSION = 'hhvm' ] || echo 'extension = mongo.so' >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini"
+  - "[ $TRAVIS_PHP_VERSION = 'hhvm' ] || echo 'extension = mongo.so' >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini"
```

Comment:
this should also be avoided on PHP 7 (as done on HHVM) as we cannot load the extension there